### PR TITLE
lava-docs: add "GitHub Actions" section

### PR DESCRIPTION
**Pull request by jroimartin**
This PR adds a new page under "User manual" to document the
"lava-action" action.

I also took the opportunity to move the "Environment variables" page
under the "Command line interface" category. I also fixed some minor
style issues.

### CODE REVIEW

File: src/github_actions.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# GitHub Actions
```
Comment (jesusfcr) [inline src/github_actions.md:1]: See my comment in https://github.com/adevinta/lava-action/pull/25#pullrequestreview-2245684332

**Review by jesusfcr — COMMENTED** [review]
I added one comment.

Also it would be nice to have the generation of the pages.json automated from `lava help` or a helper command when a new release appears.

**Review by seilagamo — APPROVED** [review]
LGTM

File: src/github_actions.md (hunk 1 above)
Comment (jroimartin) [reply src/github_actions.md:1]: It should be fixed now. PTAL.

**Review by jesusfcr — APPROVED** [review]
LGTM

**Pull request merged**